### PR TITLE
Fixes to work with template controllers

### PR DESCRIPTION
**Pull request by martingust**
@EisenbergEffect Please review this. Smaller fixes for issues that cropped up when testing `repeat` with this.

### CODE REVIEW

File: src/component-tester.js [hunk 1]
```diff
@@ -48,15 +48,19 @@ export class ComponentTester {
...
         return aurelia.start().then(a => {
           let host = document.createElement('div');
           host.innerHTML = this._html;
           document.body.appendChild(host);
           aurelia.enhance(this._bindingContext, host);
           this._rootView = aurelia.root;
           this.element = host.firstElementChild;
-          this.viewModel = this.element.au.controller.viewModel;
+          if (aurelia.root.controllers.length) {
+            this.viewModel = aurelia.root.controllers[0].viewModel;
```
Comment (martingust) [inline src/component-tester.js:62]: Only the first controller's viewModel is exposed. An improvement might be to expose all controller's viewModels. In order to get the viewModels now would be to get hold of the `aurelia` object in the configure method and grab them from there.

File: .eslintrc.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/aurelia-tools/.eslintrc.json",
+  "extends": "./../../aurelia-tools/.eslintrc.json",
```
Comment (EisenbergEffect) [inline .eslintrc.json]: Why did this need to be changed? It worked fine for me and the path seems correct. The tools should be installed. Maybe the package.json was updated and you didn't install new deps?

Comment (martingust) [inline .eslintrc.json]: It was looking for the file in wrong dir (gulp-lint), let me check this again...

**Comment by martingust** [comment]
@EisenbergEffect eslint.json is reverted.

**Comment by EisenbergEffect** [comment]
Got it. I'll release in a minute so we can get the other lib updated. Thanks!

**Pull request merged**